src/test/java/gov/usds/case_issues/db/repositories/CaseIssueRepositoryTest.java

```diff
@@ -181,6 +183,7 @@ public void saveCaseAndUpdate_withUser_auditInfoCorrect() {
 		assertInstantOrder(updatedAt, middle, false);
```

Comment:
Feel a little sketchy about this. I was able to reproduce it twice ([277](https://circleci.com/gh/usds/case-issue-api/277), [278](https://circleci.com/gh/usds/case-issue-api/278)) on Circle with this branch. But I was not able to reproduce the failure locally or on master in Circle

Reply:
It's a sort of unwinnable situation, but if a couple milliseconds of sleeping makes the ordering tests behave predictably, I'm not opposed.